ansys/dpf/core/custom_fields_container.py

```diff
@@ -1,28 +1,28 @@
 """
-Custom FieldsContainers
-=======================
-Contains inherited class of the FieldsContainer.
-Those new classes offer helpers to access data for specific usage:
-results split by body, by material...
+Custom Fields Containers
+========================
+Contains the inherited classes of the :class:`ansys.dpf.core.fields_container` class.
```

Comment:
```suggestion
Contains the inherited classes from the
:class:`FieldsContainer <ansys.dpf.core.fields_container.FieldsContainer>` class.
```

the use of `< >` allows you to have an abbreviated name by still identify the correct class.  This makes the documentation more readable as we avoid long class names.